dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/TrackedEntitiesExportControllerPostgresTest.java

```diff
@@ -0,0 +1,206 @@
+    manager.save(programStage, false);
+
+    trackedEntityType = createTrackedEntityType('A');
+    manager.save(trackedEntityType, false);
+
+    program.setTrackedEntityType(trackedEntityType);
+    manager.save(program, false);
+
+    trackedEntityAttribute = createTrackedEntityAttribute('A');
+    manager.save(trackedEntityAttribute, false);
+    TrackedEntityAttributeValue attributeValue =
+        createTrackedEntityAttributeValue('A', trackedEntity, trackedEntityAttribute);
+
+    trackedEntity = createTrackedEntity(orgUnit);
+    trackedEntity.setTrackedEntityType(trackedEntityType);
+    trackedEntity.setTrackedEntityAttributeValues(Set.of(attributeValue));
+    manager.save(trackedEntity);
+
+    injectSecurityContextUser(user);
+
+    JsonWebMessage importResponse =
+        POST(
+                "/tracker?async=false&importStrategy=UPDATE",
+                createJson(trackedEntity, trackedEntityAttribute))
+            .content(HttpStatus.OK)
+            .as(JsonWebMessage.class);
+    assertEquals(HttpStatus.OK.toString(), importResponse.getStatus());
+  }
+
+  @Test
+  void shouldGetTrackedEntityAttributeChangeLogWhenValueUpdatedAndThenDeleted() {
+    // TODO Do an e2e test instead?
+    JsonList<JsonTrackedEntityChangeLog> changeLogs =
+        GET(
+                "/tracker/trackedEntities/{id}/changeLogs?program={programUid}",
+                trackedEntity.getUid(),
+                program.getUid())
+            .content(HttpStatus.OK)
+            .getList("changeLogs", JsonTrackedEntityChangeLog.class);
+
+    JsonTrackedEntityChangeLog changeLog = changeLogs.get(0);
+    JsonUser createdBy = changeLog.getCreatedBy();
+    JsonTrackedEntityAttribute attributeChange =
+        changeLog.getChange().getTrackedEntityAttributeChange();
+    UserDetails currentUser = CurrentUserUtil.getCurrentUserDetails();
+
+    assertAll(
+        () -> assertEquals(currentUser.getUid(), createdBy.getUid()),
+        () -> assertEquals(currentUser.getUsername(), createdBy.getUsername()),
+        () -> assertEquals(currentUser.getFirstName(), createdBy.getFirstName()),
+        () -> assertEquals(currentUser.getSurname(), createdBy.getSurname()),
+        () -> assertEquals("DELETE", changeLog.getType()),
+        () ->
+            assertEquals(
+                trackedEntityAttribute.getUid(), attributeChange.getTrackedEntityAttribute()),
+        () -> assertEquals("value 3", attributeChange.getPreviousValue()),
+        () -> assertHasNoMember(attributeChange, "currentValue"));
+  }
+
+  private UserAccess userAccess() {
+    UserAccess a = new UserAccess();
+    a.setUser(user);
+    a.setAccess(AccessStringHelper.FULL);
+    return a;
+  }
+
+  private String createJson(
+      TrackedEntity trackedEntity, TrackedEntityAttribute trackedEntityAttribute) {
+    return """
+        {
+          "trackedEntities": [
+          {
+            "created": "2015-08-06T21:20:42.878",
```

Comment:
please remove fields that are not needed or will even be deliberately ignore like `created`. `created` set by the server. (its also named `createdAt` in our API)